pandas-stubs/_libs/tslibs/offsets.pyi

```diff
@@ -131,8 +132,8 @@ class BusinessHour(BusinessMixin):
         self,
         n: int = ...,
         normalize: bool = ...,
-        start: str | Collection[str] = ...,
-        end: str | Collection[str] = ...,
```

Comment:
I see no substantiation anywhere for this now removed type, seems it was copied from older code which may have been auto-generated